governance/default.go

```diff
@@ -818,18 +815,19 @@ func (g *Governance) searchCache(num uint64) (uint64, bool) {
 }
 
 func (g *Governance) ReadGovernance(num uint64) (uint64, map[string]interface{}, error) {
-	if g.ChainConfig.Istanbul == nil {
-		logger.Crit("Failed to read governance. ChainConfig.Istanbul == nil")
-	}
-	blockNum := CalcGovernanceInfoBlock(num, g.epochWithFallback())
+	// if num is zero, epochWithFallback() will Crit
+	// so epoch should be set to any value greater than zero
```

Comment:
Please update the comments according to the current `epochWithFallback()`. Thank you